components/reminder/ReminderForm.js

```diff
@@ -1,44 +1,64 @@
-import { useState } from "react";
+import { useEffect, useState } from "react";
 import { useRouter } from "next/router";
 import useSWR from "swr";
 import styled from "styled-components";
 import { CheckCircleIcon, XCircleIcon } from "@phosphor-icons/react";
 import Link from "next/link";
 import { toast } from "react-toastify";
 
-export default function ReminderForm({ userId }) {
+export default function ReminderForm({ userId, reminderId }) {
   const router = useRouter();
-  const { data: plants, error } = useSWR(
+  const { data: plants, error: plantsError } = useSWR(
     userId ? `/api/user/${userId}/owned` : null
   );
+  const { data: reminder, error: reminderError } = useSWR(
+    reminderId ? `/api/user/${userId}/reminders/${reminderId}` : null
+  );
 
+  const quickActions = ["Water", "Fertilise", "Repot"];
   const [title, setTitle] = useState("");
+  const [plantId, setPlantId] = useState("");
```

Comment:
```suggestion
  const [plantId, setPlantId] = useState(reminder?.plantId ?? "");
```
-> You can get rid of the useEffect.